### PR TITLE
Lang tag for collections/admin/igsnmapper.php

### PR DESCRIPTION
**Pull request by NikitaSalikov**
# Changes made:

- Internationalized text

# Pull Request Checklist:

# Pre-Approval

- [x] There is a description section in the pull request that details what the proposed changes do. It can be very brief if need be, but it ought to exist.
- [ ] Hotfixes should be branched off of the `master` branch and **squash and merged** back into the `master` branch.
- [x] Features and backlog bugs should be merged into the `Development` branch, **NOT** `master`
- [x] All new text is preferably internationalized (i.e., no end-user-visible text is hard-coded on the PHP pages), and the [spreadsheet tracking internationalizations](https://docs.google.com/spreadsheets/d/133fps9w2pUCEjUA6IGCcQotk7dn9KvepMXJ2IWUZsE8/edit?usp=sharing) has been updated either with a new row or with checkmarks to existing rows.
- [ ] There are no linter errors
- [x] New features have responsive design (i.e., look aesthetically pleasing both full screen and with small or mobile screens)
- [ ] [Symbiota coding standards](https://docs.google.com/document/d/1-FwCZP5Zu4f-bPwsKeVVsZErytALOJyA2szjbfSUjmc/edit?usp=sharing) have been followed
- [ ] If any files have been reformatted (e.g., by an autoformatter), the reformat is its own, separate commit in the PR
- [x] Comment which GitHub issue(s), if any does this PR address:
https://github.com/orgs/BioKIC/projects/6/views/14?pane=issue&itemId=53327784
- [ ] If this PR makes any changes that would require additional configuration of any Symbiota portals outside of the files tracked in this repository, make sure that those changes are detailed in [this document](https://docs.google.com/document/d/1T7xbXEf2bjjm-PMrlXpUBa69aTMAIROPXVqJqa2ow_I/edit?usp=sharing).

# Post-Approval

- [x] It is the code author's responsibility to merge their own pull request after it has been approved
- [x] If this PR represents a merge into the `Development` branch, remember to use the **squash & merge** option
- [ ] If this PR represents a merge from the `Development` branch into the master branch, remember to use the **merge** option
- [ ] If this PR represents a hotfix into the `master` branch, a subsequent PR from `master` into `Development` should be made **merge** option (i.e., no squash).
- [ ] If the dev team has agreed that this PR represents the last PR going into the `Development` branch before a tagged release (i.e., before an imminent merge into the master branch), make sure to notify the team and [lock the `Development` branch](https://github.com/BioKIC/Symbiota/settings/branches) to prevent accidental merges while QA takes place. Follow the release protocol [here](https://github.com/BioKIC/Symbiota/blob/master/docs/release-protocol.md).
- [x] Don't forget to delete your feature branch upon merge. Ignore this step as required.

Thanks for contributing and keeping it clean!

### CODE REVIEW

File: collections/admin/igsnmapper.php [hunk 1]
```diff
@@ -169,10 +173,10 @@ function verifyGuidForm(f){
 include($SERVER_ROOT.'/includes/header.php');
 ?>
 <div class='navpath'>
-	<a href="../../index.php">Home</a> &gt;&gt;
-	<a href="../misc/collprofiles.php?collid=<?php echo htmlspecialchars($collid, HTML_SPECIAL_CHARS_FLAGS); ?>&emode=1">Collection Management</a> &gt;&gt;
-	<a href="igsnmanagement.php?collid=<?php echo htmlspecialchars($collid, HTML_SPECIAL_CHARS_FLAGS); ?>">IGSN GUID Management</a> &gt;&gt;
-	<b>IGSN Mapper</b>
+	<a href="../../index.php"><?php echo $LANG['HOME'] ?></a> &gt;&gt;
+	<a href="../misc/collprofiles.php?collid=<?php echo htmlspecialchars($collid, HTML_SPECIAL_CHARS_FLAGS); ?>&emode=1"><?php echo $LANG['CALL_MANAGE'] ?></a> &gt;&gt;
```
Comment (themerekat) [inline collections/admin/igsnmapper.php]: Not super important, but I would change this to COLL_MANAGE

File: content/lang/collections/admin/igsnmapper.en.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+------------------
+*/
+
+$LANG['IGSN_GUID_MAPPER'] = 'IGSN GUID Mapper';
+$LANG['VALUEHERE'] = 'INSERTVALUE HERE!!!!';
+$LANG['ENTER_VALID'] = 'Please enter valid SESAR username and password';
+$LANG['MUST_BE_EMAIL'] = 'SESAR username must be an email address';
+$LANG['SELECT_REG'] = 'Select a registration method';
+$LANG['IGSN_NOT_GENERATED'] = 'In-house IGSN Generation selected but IGSN seed not generated (contact administrator)';
+$LANG['CALL_MANAGE'] = 'Collection Management';
```
Comment (themerekat) [inline content/lang/collections/admin/igsnmapper.en.php]: COLL_MANAGE

File: content/lang/collections/admin/igsnmapper.es.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+------------------
+*/
+
+$LANG['IGSN_GUID_MAPPER'] = 'Asignador GUID IGSN';
+$LANG['VALUEHERE'] = '¡¡¡INSERTVALOR AQUÍ!!!!';
+$LANG['ENTER_VALID'] = 'Por favor, introduzca un nombre de usuario y una contraseña válidos de SESAR';
+$LANG['MUST_BE_EMAIL'] = 'El nombre de usuario de SESAR debe ser una dirección de correo electrónico';
+$LANG['SELECT_REG'] = 'Seleccione un método de registro';
+$LANG['IGSN_NOT_GENERATED'] = 'Generación de IGSN interna seleccionada pero semilla de IGSN no generada (contacte al administrador)';
+$LANG['CALL_MANAGE'] = 'Gestión de colecciones';
```
Comment (themerekat) [inline content/lang/collections/admin/igsnmapper.es.php]: COLL_MANAGE

File: content/lang/collections/admin/igsnmapper.fr.php [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+------------------
+*/
+
+$LANG['IGSN_GUID_MAPPER'] = 'Mappeur de GUID IGSN';
+$LANG['VALUEHERE'] = 'INSÉRER LA VALEUR ICI!!!!';
+$LANG['ENTER_VALID'] = 'Veuillez saisir un nom d\'utilisateur et un mot de passe SESAR valides';
+$LANG['MUST_BE_EMAIL'] = 'Le nom d\'utilisateur SESAR doit être une adresse e-mail';
+$LANG['SELECT_REG'] = 'Sélectionnez une méthode d\'inscription';
+$LANG['IGSN_NOT_GENERATED'] = 'Génération IGSN interne sélectionnée mais graine IGSN non générée (contacter l\'administrateur)';
+$LANG['CALL_MANAGE'] = 'Gestion des collections';
```
Comment (themerekat) [inline content/lang/collections/admin/igsnmapper.fr.php]: COLL_MANAGE

File: collections/admin/igsnmapper.php [hunk 5]
```diff
@@ -205,65 +209,65 @@ function verifyGuidForm(f){
 			<form id="guidform" name="guidform" action="igsnmapper.php" method="post" onsubmit="return verifyGuidForm(this)">
 				<input type="hidden" name="collid" value="<?php echo $collid; ?>" />
 				<fieldset>
-					<legend>IGSN Registration Control Panel</legend>
-					<p>Register IGSNs via a manual submission of data file or automaticly using <a href="http://www.geosamples.org/interop" target="_blank">SESAR API Web Services</a></p>
+					<legend><?php echo $LANG['IGSN_REG_PANEL'] ?></legend>
+					<p><?php echo $LANG['REG_IGSN'] . " "?><a href="http://www.geosamples.org/interop" target="_blank">SESAR API Web Services</a></p>
```
Comment (themerekat) [inline collections/admin/igsnmapper.php]: Oops, missing a lang tag for SESAR API Web Services

**Review by themerekat — CHANGES_REQUESTED** [review]

**Comment by Atticus29** [comment]
@NikitaSalikov , is this one ready for re-review?

**Comment by NikitaSalikov** [comment]
> @NikitaSalikov , is this one ready for re-review?

Yes, it is!

**Review by themerekat — APPROVED** [review]

**Pull request merged**